pkg/synthetictests/duplicated_events_test.go

```diff
@@ -309,22 +305,21 @@ func TestKnownBugEvents(t *testing.T) {
 			events = append(events,
 				monitorapi.Interval{
 					Condition: monitorapi.Condition{Message: test.message},
-					From:      time.Unix(1, 0),
-					To:        time.Unix(1, 0)},
+					From:      time.Unix(1, 0).In(time.UTC),
+					To:        time.Unix(1, 0).In(time.UTC)},
 			)
 			evaluator.platform = test.platform
 			evaluator.topology = test.topology
 
 			junits := evaluator.testDuplicatedEvents("events should not repeat", false, events, nil, true)
-			if len(junits) < 1 {
-				t.Fatal("didn't get junit for duplicated event")
-			}
-			if test.match && !strings.Contains(junits[0].FailureOutput.Output, "1 events with known BZs") {
-				t.Fatalf("expected case to match, but it didn't: %s", test.name)
-			}
-			if !test.match && strings.Contains(junits[0].FailureOutput.Output, "1 events with known BZs") {
-				t.Fatalf("expected case to not match, but it did: %s", test.name)
+			assert.GreaterOrEqual(t, len(junits), 1, "didn't get junit for duplicated event")
```

Comment:
Your expected and actual are reversed for many of your asserts, it will work, but the output will be confusing. Assert library wants expected value first, then actual.

Reply:
In this case `assert.GreaterOrEqual` is looking for param1 >= param2. I think the order is correct.

Reply:
My mistake that is ok with that func then.